free-courses-hi.md

```diff
@@ -46,3 +46,9 @@
 ### Python
 
 * [Python Tutorial For Beginners | Hindi (With Notes)](https://www.youtube.com/playlist?list=PLu0W_9lII9agICnT8t4iYVSZ3eykIAOME) - Harry
+
+### Dynamic Programming
+
+* [Dynamic Programming in Hindi| Detailed Analysis from recursive equation to Bottom Up]  (https://www.youtube.com/playlist?list=PLeF0b8iqbx4mogykbd82-HY9Y1-JS9MDr) Shashank Jha
```

Comment:
Hey, and welcome to free programming books! :wave:

* Please use the title from the source, don't create your own.
* Markdown links should not have a space between the ] and (
* There should be a hyphen with spaces around before the author/vendor.
* Not sure if dynamic programming needs to be its own section, but could you please add a heading for it while it's here?

```suggestion
* [Dynamic Programming in Hindi](https://www.youtube.com/playlist?list=PLeF0b8iqbx4mogykbd82-HY9Y1-JS9MDr) - Shashank Jha
```

Reply:
I didn't notice until now, could you please also ensure new sections have two empty lines above them.